openapi/components/schemas/CreditMemos/CreditMemoProperties.yaml

```
@@ -0,0 +1,176 @@
+id:
+  description: Credit memo ID.
+  readOnly: true
+  allOf:
+    - $ref: ../ResourceId.yaml
+invoiceId:
+  description: ID of the invoice on which the credit memo is issued.
+  nullable: true
+  allOf:
+    - $ref: ../ResourceId.yaml
+customerId:
+  $ref: ../CustomerId.yaml
+number:
+  description: An auto-incrementing number based on the sequence of credit memos for any particular customer.
+  readOnly: true
+  type: integer
+allocations:
+  type: object
+  description: Allocations reduce unused amount of the credit memo.
+  properties:
+    transactions:
+      type: array
+      description: |-
+        List of transactions (typically refunds, credits, and chargebacks) that are allocated to reducing unused amount of the credit memo.
```

suggestion: possible typo
```suggestion
        List of transactions (typically refunds, credits, and chargebacks) that are allocated to reduce the unused amount of a credit memo.
```